package/efs/rancher-efs

```diff
@@ -214,14 +220,21 @@ mountdest() {
 
     get_meta_data
 
-    efsDNSName=${EC2_AVAIL_ZONE}.${OPTS[fsid]}.efs.${EC2_REGION}.amazonaws.com
+    local efsMountIP="${OPTS[mountTargetIp]}"
+    # Amazon takes a while to make the mount target resolvable via DNS, so...
+    while [ "$efsMountIP" == "" ]; do
```

Comment:
Also, regarding dig's exit code - it is 0 even if no records are returned. The DNS server is functioning, but the A record isn't yet propogated, hence the loop.